eslint.config.js

```diff
@@ -1,44 +1,41 @@
-import { fixupConfigRules } from '@eslint/compat';
-import pluginJs from '@eslint/js';
-import * as reactQuery from '@tanstack/eslint-plugin-query';
-import pluginImport from 'eslint-plugin-import';
-import pluginReactConfig from 'eslint-plugin-react/configs/recommended.js';
```

Comment:
I belive you accidentally delete eslint-plugin-react

Reply:
Yup. Fixed. According to the eslint-plugin-react docs.